src/main/java/com/amazon/dlic/auth/ldap/LdapUser.java

```diff
@@ -45,6 +48,12 @@ public LdapUser(
         attributes.putAll(extractLdapAttributes(originalUsername, userEntry, customAttrMaxValueLen, allowlistedCustomLdapAttrMatcher));
     }
 
+    public LdapUser(StreamInput in) throws IOException {
+        super(in);
+        userEntry = null;
```

Comment:
Curious, is it necessary to set the `userEntry` to null here? I see that this is a `private final transient` field and I'm not sure exactly what it means to be `transient`.

Reply:
I see, `transient` means it does not get serialized.